src/frontend/hooks/useHentBehandling.ts

```diff
@@ -8,7 +8,7 @@ export const useHentBehandling = (
     behandlingId: string
 ): {
     hentBehandlingCallback: () => void;
-    behandling: Ressurs<Behandling>;
+    behandling: Ressurs<Behandling> /** Hvorfor er denne pakket inn slik ? */;
```

Comment:
Api-et returnerer en Ressurs med status:
- FEILET
- HENTER
- IKKE_HENTET
- IKKE_TILGANG
- SUKSESS
- FUNKSJONELL_FEIL

Hvis statusen er Suksess så vil du ha et dataobjekt med typen som er definert i `<...>`
For de andre statusene har du ikke noe dataobjekt. 

Slik sett kan vi sjekke på Ressurs' status hvorvidt kallet har gått bra og vi skal bruke dataene, om vi skal vise en spinner eller om vi skal vise en feilmelding. 

Alle http-kall som feiler uten å returnere Ressurs blir wrappet til ressurs med status `Feilet` via `AppContext.tsx` og `axios.ts` med funksjonen `håndterRessurs`